### PR TITLE
reformat

### PR DESCRIPTION
**Pull request by maxmarzolf**

### CODE REVIEW

File: game.py [hunk 1]
```diff
@@ -131,7 +131,7 @@ def combat_series(self):
                             print(f"The computer hit you for {computer_attack_damage} damage!")
                             sleep(1)
```
Comment (maxmarzolf) [inline game.py:132]: consider increasing sleep time

**Review by maxmarzolf — COMMENTED** [review]
mild reformat

File: game.py (hunk 1 above)
Comment (Brett-BI) [reply game.py:132]: ... or creating a method to let the user know that the application is 'thinking'?

**Review by Brett-BI — CHANGES_REQUESTED** [review]
See conversations?

File: game.py (hunk 1 above)
Comment (maxmarzolf) [reply game.py:132]: sounds good.

**Pull request merged**